src/ig/IGExporter.ts

```diff
@@ -405,6 +407,76 @@ export class IGExporter {
     });
   }
 
+  /**
+   * Adds any user provided resource files
+   * This includes definitions in:
+   * capabilities, models, operations, profiles, resources, vocabulary, examples
+   *
+   * @param {string} igPath - the path where the IG is exported to
+   */
+  private addPredefinedResources(igPath: string): void {
+    const pathEnds = [
+      'capabilities',
+      'models',
+      'operations',
+      'profiles',
+      'resources',
+      'vocabulary',
+      'examples' // Must come last in case examples are of other resources
+    ];
```

Comment:
Do we want to include `extensions` in this list here? It's a supported folder and you mentioned it in the PR description, but right now, it won't get copied over.

Reply:
Oh most definitely we do!